subsys/bluetooth/host/Kconfig.l2cap

```diff
@@ -44,6 +44,13 @@ config BT_L2CAP_TX_MTU
 	help
 	  Maximum L2CAP MTU for L2CAP TX buffers.
 
+config BT_L2CAP_TX_USER_DATA_SIZE
+	int
+	default 8
```

Comment:
Same here

Reply:
it's not user-settable

Reply:
actually this is a mistake, I just didn't save the file. Only the conn one is valid.